messages/event_manager.go

```diff
@@ -99,8 +99,9 @@ func (em *eventManager) close() {
 	em.subscriptionsLock.Lock()
 	defer em.subscriptionsLock.Unlock()
 
-	for _, subscription := range em.subscriptions {
```

Comment:
Could you also take care about https://github.com/0xPolygon/go-ibft/pull/38 ? (merge main, finish task)

Reply:
I merged latest `develop` and fix conflict